### PR TITLE
:zap: Release v2.0.1

### PR DESCRIPTION
**Pull request by flovilmart**
This PR does a few things:

1. fix the issue with the headers, when running with an older parse-server
2. updates the documentations for Cloud Code for parse-server 3.0. This is not ideal that the Cloud Code docs isn't in parse-server itself, but that's how it is, perhaps we could do something about it like a like or I dunno

### CODE REVIEW

**Comment by flovilmart** [comment]
@miconblog can you have a look there? as you went through the upgrade process, I'd like to release the fixes in the 2.0.1 as well as the upgrading guide.

**Comment by miconblog** [comment]
yeb! i am already reading this guide. and migrating my project

**Comment by flovilmart** [comment]
Let me know if there's anything missing or that you believe we should put :)

File: src/CloudCode.js [hunk 1]
```diff
@@ -16,19 +18,19 @@
  *
  * If you want to use afterDelete for a predefined class in the Parse JavaScript SDK (e.g. {@link Parse.User}), you should pass the class itself and not the String for arg1.
  * ```
- * Parse.Cloud.afterDelete('MyCustomClass', (request) => {
+ * Parse.Cloud.afterDelete('MyCustomClass', async (request) => {
```
Comment (miconblog) [inline src/CloudCode.js]: i just curios about this CloudCode. Is it work in browser now? or is your long-team plan?

Comment (flovilmart) [reply src/CloudCode.js]: No it's in the server, but that's easier to have the whole cloud code API documentation with the rest of the JS SDK, we may move it at a later point if we generate a bigger API docs for parse-server.

Comment (miconblog) [reply src/CloudCode.js]: okay, i see i just confused :)

File: src/CloudCode.js [hunk 2]
```diff
@@ -62,19 +64,19 @@
  *
  * If you want to use beforeDelete for a predefined class in the Parse JavaScript SDK (e.g. {@link Parse.User}), you should pass the class itself and not the String for arg1.
  * ```
- * Parse.Cloud.beforeDelete('MyCustomClass', (request, response) => {
```
Comment (miconblog) [inline src/CloudCode.js]: is it remove response in beforeDelete? 
i was used it before like this:
```
Parse.Cloud.beforeDelete('RedBookNote', async function (request, response) {
    response.success();
});

```

Comment (flovilmart) [reply src/CloudCode.js]: In parse-server 3.0 (the the docs update is for parse-server 3.0)

Comment (flovilmart) [reply src/CloudCode.js]: I understand this leads to a lot of confusion, I'll bring in the CLoud Code docs in parse-server instead. And just add the link here. I'll revert cloud code docs for now :)

Comment (miconblog) [reply src/CloudCode.js]: goood :)

**Pull request merged**